test_build.py

```diff
@@ -70,3 +71,40 @@
 
 from skimage.io import imread
 print("skimage ok")
+
+import os
+import threading
+from http.server import BaseHTTPRequestHandler, HTTPServer
+from google.cloud import bigquery
+
+HOSTNAME = "127.0.0.1"
+PORT = 8000
+URL = "http://%s:%s" % (HOSTNAME, PORT)
+
+header_found = False
+
+class HTTPHandler(BaseHTTPRequestHandler):
+    def do_HEAD(s):
+        s.send_response(200)
+
+    def do_GET(s):
+        global header_found
```

Comment:
Nit:  you could have a `fake_bq_called` as well, so if this test fails, we know more precisely what the issue is (never called or called but no header).